$DragonFly: src/lib/libpam/patches/openpam_configure.c.patch,v 1.1 2005/07/13 12:34:21 joerg Exp $

--- openpam_configure.c.orig	2005-07-07 19:43:40.000000000 +0200
+++ openpam_configure.c	2005-07-07 19:44:09.000000000 +0200
@@ -69,7 +69,8 @@
 match_word(const char *str, const char *word)
 {
 
-	while (*str && tolower(*str) == tolower(*word))
+	while (*str &&
+	       tolower((unsigned char)*str) == tolower((unsigned char)*word))
 		++str, ++word;
 	return (*str == ' ' && *word == '\0');
 }
